modules/accessing-metrics-outside-cluster.adoc

```diff
@@ -8,44 +8,55 @@
 
 Learn how to query Prometheus statistics from the command line when monitoring your own services. You can access monitoring data from outside the cluster with the `thanos-querier` route.
 
+This access only supports using a Bearer Token for authentication.
+
 .Prerequisites
 
-* You deployed your own service, following the _Enabling monitoring for user-defined projects_ procedure.
+* You have deployed your own service, following the "Enabling monitoring for user-defined projects" procedure.
+* You are logged in to an account that has permission to access the Thanos Querier API.
++
+[NOTE]
+====
+To access the `thanos-querier` API route, the `cluster-monitoring-view` cluster role must be bound to the account.
```

Comment:
bound `cluster-monitoring-view` cluster role to user testuser-11 by admin
```
$ oc adm policy add-cluster-role-to-user cluster-monitoring-view testuser-11
clusterrole.rbac.authorization.k8s.io/cluster-monitoring-view added: "testuser-11"
```
user testuser-11 failed to get thanos-querier route
```
# HOST=$(oc -n openshift-monitoring get route thanos-querier -ojsonpath={.spec.host})
Error from server (Forbidden): routes.route.openshift.io "thanos-querier" is forbidden: User "testuser-11" cannot get resource "routes" in API group "route.openshift.io" in the namespace "openshift-monitoring"
```

Reply:
@simonpasquier - Can you PTAL at @juzhao's comment and let me know how I should update the information about what permissions are required to get the Thanos Querier route?

Reply:
@juzhao  is correct and I alluded to it in my first review (e.g. "If your account does not have permissions to read routes, a cluster administrator can provide the URL for the route."). Can we move the retrieval of the Thanos Querier route to the pre-requisites?

Reply:
@juzhao and @simonpasquier - PTAL at the changes I've made to the PR to address the issues you've mentioned. Thanks!